scrapper.py

```diff
@@ -1,6 +1,20 @@
 """
 Scrapper implementation
 """
+import json
+import re
+import shutil
+import requests
+from bs4 import BeautifulSoup
+from pathlib import Path
+from random import randint
+from time import sleep
+from datetime import datetime
+from constants import ASSETS_PATH
+from constants import CRAWLER_CONFIG_PATH
+from constants import HEADERS
+from constants import DOMAIN_NAME
+from core_utils.article import Article
 
```

Comment:
Correct import order is as follows: standard libraries -- external third-party libraries -- local modules 
Within each of the three blocks of imports, alphabetical order is recommended